mypy.ini

```diff
@@ -0,0 +1,37 @@
+[mypy]
```

Comment:
Hate to do this to you in the midnight hour, I didn't notice that this was a new file until now. Can you please implement this ala-poetry in `pyproject.toml`?

Reply:
i've moved the config to `pyproject.toml`, but i'm not totally happy with it. running `poetry run mypy` results in a bunch of errors in the `_vendor` module, even if it's added to the 'whitelist'. not 100% sure why that is. potentially have to silence the import errors? will try and find some time in the next few days for a more thorough investigation.

Reply:
Instead of using a module whitelist (that won't work as the modules in `_vendor` are not part of the `poetry.core` namespace), have you tried a file-glob exclude in `pyproject.toml`?

https://mypy.readthedocs.io/en/stable/config_file.html#confval-exclude

Reply:
does https://github.com/python-poetry/poetry-core/pull/199#discussion_r748913374 help?

Reply:
> 
> 
> does [#199 (comment)](https://github.com/python-poetry/poetry-core/pull/199#discussion_r748913374) help?

It was exactly what I meant!